### ISSUE TITLE
Import existing files to project

### ISSUE THREAD

**Issue by august0815** [Valama/valama#8]

**Comment by agronick** [comment]
Yes, I need this. Is there any way to accomplish this. Its 3 years old. It does not even detect when new files are added to the directories. There is no way to import existing files either. How do work with existing code?

This is such a shame since this is the only program that handles autocompletion this well.

**Comment by LSenf** [comment]
Hi, dirty workaround: edit the vlp project file manually.
As for the future, valamang supports this properly. I can use some help there by the way, see valamang issues :)

**Comment by agronick** [comment]
Sorry, I haven't heard of valamang. I'm pretty new to Vala. Is it another IDE?

**Comment by inizan-yannick** [comment]
@agronick Valama Next Generation. see valamang branch : https://github.com/Valama/valama/tree/valamang

**Comment by agronick** [comment]
Oh I tried that. No offense but it needs a lot of work. I'd like to help but I'm new to this.  Are there any easy tasks that involve coding that you think I could assist with?

**Comment by inizan-yannick** [comment]
draw UI for Valama  with Glade and submit work ? :)

**Comment by LSenf** [comment]
You can have a look at the issues labeled "call for contribution". If you have any questions about the code, please ask :)
One rather important issue is getting a build system to work. The basic infrastructure is there, the actual implementations are still missing.
